packages/vue-dot/src/patterns/FormBuilder/tests/FormBuilder.spec.ts

```diff
@@ -48,4 +75,20 @@ describe('FormBuilder', () => {
 
 		expect(html(wrapper)).toMatchSnapshot();
 	});
+
+	it('renders form question correctly', () => {
```

Comment:
```suggestion
	it('renders correctly with a specified default layout', () => {
```
Je pense que c'est plus explicite non ? :thinking: